### ISSUE TITLE
Resizable panels

### ISSUE THREAD

**Issue by riyasvaliya** [elkozmon/zoonavigator#67]
Is there any way to make the panels resizable (as in allow dragging of the splitter to expand the zNode list area)?  We have zNodes with long names and common prefixes, so currently it is impossible to differentiate the nodes since the names (upto what is displayed) is the same

**Comment by elkozmon** [comment]
Hi. 

I'm not aware if there's such functionality in the UI library I use, but I will have a look (may take some time as I'm terribly busy at the moment).

But still, there should be a tooltip with the full name of the z-node when you hover over it, isn't there?

**Comment by riyasvaliya** [comment]
> Hi.
> 
> I'm not aware if there's such functionality in the UI library I use, but I will have a look (may take some time as I'm terribly busy at the moment).
> 
> But still, there should be a tooltip with the full name of the z-node when you hover over it, isn't there?

Yup, the tooltip on hover is present. Will make do with that for now until you get the time. Thanks :+1:
